apply_factors_wrangler.py

```diff
@@ -108,15 +110,20 @@ def lambda_handler(event, context):
             on="responder_id",
         )
         logger.info("Successfully merged previous period data with non-responder df")
+
+        # filter factors df to only get current period
+        factors_dataframe = factors_dataframe[
+            factors_dataframe['period'] == int(current_period)]
```

Comment:
Shouldn't have a hardcoded period_column name and should use the RuntimeVariable instead.